tests/framework/installer/ceph_manifests.go

```diff
@@ -2321,11 +2321,11 @@ rules:
   - update
   - delete
 - apiGroups:
-  - k8s.cni.cncf.io
+  - batch
```

Comment:
Adding batch, cronjobs, delete wasn't part of this PR in master, right? Was this change picked up from the merge conflict? Seems like we don't need to add it here? But I didn't look very deep...

Reply:
It was picked up from merge conflict.
I will look into it and confirm if we need the changes or not.

Reply:
Compared with https://github.com/rook/rook/blob/release-1.5/tests/framework/installer/ceph_manifests.go#L2323, looks like these changes are not required.